Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyColorCtrl.cpp

```diff
@@ -400,7 +400,12 @@ namespace AzToolsFramework
             {
                 EBUS_EVENT(PropertyEditorGUIMessages::Bus, RequestWrite, newCtrl);
             });
-        // note:  Qt automatically disconnects objects from each other when either end is destroyed, no need to worry about delete.
+        connect(newCtrl, &PropertyColorCtrl::editingFinished, this, [newCtrl]()
+            {
+                AzToolsFramework::PropertyEditorGUIMessages::Bus::Broadcast(
+                    &PropertyEditorGUIMessages::OnEditingFinished, newCtrl);
+            });
```

Comment:
Interesting....how were color pickers working correctly before without this change?

I'm trying to understand why this needs to be added now.

Reply:
Color pickers are using the AZColorPropertyHandler which already had the OnEditingFinished hooked up. It's being added now to Vector3ColorPropertyHandler so we have all our AzToolFramework property handlers follow the expected pattern of sending both RequestWrite during editing and OnEditingFinished at the end of editing.